KittiOdom2Bag:
  ros__parameters:
    sequence: 02
    data_dir: '/path/to/kitti/dataset/'
    odom_dir: '/path/to/kitti/dataset/'
    bag_dir : './odom_bag'
    odom : True
    raw_data:
      dir: 'raw_kitti/'
      date: '2011_10_03'
      drive: '0042'
      start_frame: 0
      end_frame: 1100
# Nr.     Sequence name     Start   End
# ---------------------------------------
# 00: 2011_10_03_drive_0027 000000 004540
# 01: 2011_10_03_drive_0042 000000 001100
# 02: 2011_10_03_drive_0034 000000 004660
# 03: 2011_09_26_drive_0067 000000 000800
# 04: 2011_09_30_drive_0016 000000 000270
# 05: 2011_09_30_drive_0018 000000 002760
# 06: 2011_09_30_drive_0020 000000 001100
# 07: 2011_09_30_drive_0027 000000 001100
# 08: 2011_09_30_drive_0028 001100 005170
# 09: 2011_09_30_drive_0033 000000 001590
# 10: 2011_09_30_drive_0034 000000 001200